src/components/navbar.jsx

```diff
@@ -1,41 +1,57 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import { ImUser } from 'react-icons/im';
 
 export default function Header() {
   const [data, setData] = useState({
     render: true,
-    class: 'links',
+    class: 'links container',
   });
 
   function toggle() {
     const { render } = data;
     if (render === false) {
       setData(() => ({
         render: true,
-        class: 'links',
+        class: 'links container',
       }));
     } else {
       setData(() => ({
         render: false,
-        class: 'links hide',
+        class: 'links container hide',
       }));
     }
   }
-
   return (
     <header className="navbar">
-      <h1 className="header">Bookstore</h1>
-      <nav className="navdiv">
-        {(data.render) ? <button type="button" className="navbutton" onClick={toggle}>menu</button> : <button type="button" className="navbutton" onClick={toggle}>close</button> }
-        <ul className={`${data.class}`}>
-          <li>
-            <button type="button" className="linkbutton" onClick={toggle}><Link to="/" className="link">Home</Link></button>
-          </li>
-          <li>
-            <button type="button" className="linkbutton" onClick={toggle}><Link to="Categories" className="link">Categories</Link></button>
-          </li>
-        </ul>
-      </nav>
+      <div className="navwrap container">
+        <h1 className="header">Bookstore CMS</h1>
+        <nav className="navdiv container">
+          <ul className={data.class}>
+            <li>
+              <button type="button" className="linkbutton" onClick={toggle}><Link to="/" className="link">Home</Link></button>
+            </li>
+            <li>
+              <button type="button" className="linkbutton" onClick={toggle}><Link to="Categories" className="link">Categories</Link></button>
+            </li>
+          </ul>
+        </nav>
```

Comment:
- [ ] I noticed that your navbar elements are not in Uppercase like on the zeplin desing . Please fix it in order to make your project looks like expected 👍 

#### You design
![bookstore](https://github.com/vikas924/bookstore/assets/6615394/9213f756-6582-49a3-8b36-7037a4a334ac)

#### Expected design
![bookstore](https://github.com/vikas924/bookstore/assets/6615394/5b87d32e-d251-48d7-84e7-40a345fe000d)